lib/topology/injection.py

```diff
@@ -155,7 +155,13 @@ def subfolders(search_path):
                         result[filename][node] = {}
 
                     for attribute, value in modifier['attributes'].items():
-                        result[filename][node][attribute] = value
+                        if isinstance(value, dict):
```

Comment:
I agree with @carlos-jenkins here, I'll also prefer to keep topology element (nodes, ports and links) attributes "flat" (without any nesting structure, like a dictionary). @Chornei, do you have a particular need for attribute injection?